lib/package.rb

```diff
@@ -167,9 +131,7 @@ def self.system(*args)
     # Add -j arg to build commands.
     cmd_args.sub(/\b(?<=make)(?=\b)/, " -j#{CREW_NPROC}") unless cmd_args =~ /-j\s*\d+/
     # Escape special bash characters.
-    cmd_args.gsub!('"','\\\\"')
-    cmd_args.gsub!('$','\\\\$')
-    cmd_args.gsub!('`','\\\\`')
+    cmd_args.gsub!(/("|\$|`)/, '\\\\\1')
```

Comment:
Can we just come up with something better than escaping?  This is extremely hackish.

Reply:
Some possible solutions:
- Create an `IO` object based on the command and pass it to bash through stdin
- Pass the command as a separate parameter (`Kernel.system 'bash', '-c', '-e', @command`) so that `/bin/bash` will not be called before `bash`
- use heredoc:
```shell
system <<~BASH
  bash -e -c <<'CMD'
  #{@command}
  CMD
BASH
```

Reply:
@satmandu Which one do you like better?

Reply:
> @satmandu Which one do you like better?

I think the arguments to `bash -e -c` need to be passed in quotes, no?

Reply:
I was just thinking do we even need bash?  The directory this is run in has exec permissions so it could (should?) be able to run without bash, right?

Reply:
Actually `bash` is not necessary, but it allows us to use some extra expressions in `bash` like regex